manual/mapper/daos/defaultnullsavingstrategy/README.md

```diff
@@ -0,0 +1,33 @@
+## DefaultNullSavingStrategy
+
+If you don't specify `nullSavingStrategy` parameter on the method level you can suppress 
+defaults by using `@DefaultNullSavingStrategy` annotation.
+
+This annotation works at the [@Dao] level:
+```java
+@Dao
+@DefaultNullSavingStrategy(nullSavingStrategy = NullSavingStrategy.SET_TO_NULL)
+public interface DefaultNullStrategyDao {
+  // dao methods
+}
+
+```
+
+Strategies when using `@DefaultNullSavingStrategy` with method level `nullSavingStrategy` on annotations: 
+[@Insert](../insert/), [@Query](../query/), [@SetEntity](../setentity/) or
+[@Update](../update/):
+
+* If you specify `@DefaultNullSavingStrategy` at the [@Dao] level and set `nullSavingStrategy` parameter on one of the 
+  [@Update](../update/), [@Insert](../insert/), [@Query](../query/) or [@SetEntity](../setentity./)
+  explicitly, the method level annotation value will be taken.
+
+* If you specify `@DefaultNullSavingStrategy` at the [@Dao] level and do not set `nullSavingStrategy` parameter on one of the 
```

Comment:
These next two paragraphs don't add much as I think it should be apparent that if you only use one of the ways to configure null saving, the one that is used is used.

Reply:
The method level annotations have `NullSavingStrategy nullSavingStrategy() default NullSavingStrategy.DO_NOT_SET;` value. When the user is using it will have a default set.
Don't you think that some of our users may think that this unset default can take precedence?
I think that there is a small chance that someone may think like this so maybe it is better to have it explicitly documented?